resources/public/api/conf/1.0/examples/BonusPaymentsuperseded.post.json

```diff
@@ -0,0 +1,22 @@
+{
+  "periodStartDate": "2018-04-06",
+  "periodEndDate": "2018-05-05",
+  "inboundPayments": {
+    "newSubsForPeriod": 0.00,
+    "newSubsYTD": 0.00,
+    "totalSubsForPeriod": 0.00,
+    "totalSubsYTD": 0.00
+  },
+  "bonuses": {
+    "bonusDueForPeriod": 0.00,
+    "totalBonusDueYTD": 0.00,
+    "claimReason": "Regular Bonus"
+  },
+  "supersede" : {
+    "automaticRecoveryAmount":300.00,
```

Comment:
I think we could do better with the field names, they don't seem intuitive to me (may be just me?). I'd also be inclined to drop 'superseded' from the field names as they're already in a 'supersede' object.